src/snowflake/connector/errors.py

```diff
@@ -66,9 +68,9 @@ def __init__(
             if self.sqlstate != "n/a":
                 if not already_formatted_msg:
                     if logger.getEffectiveLevel() in (logging.INFO, logging.DEBUG):
-                        self.msg = f"{self.errno:06d} ({self.sqlstate}): {self.sfqid}: {self.msg}"
+                        self.msg = f"{self.errno:06d} ({self.sqlstate}): {self.sfqid}: {self.query}: {self.msg}"
                     else:
-                        self.msg = f"{self.errno:06d} ({self.sqlstate}): {self.msg}"
+                        self.msg = f"{self.errno:06d} ({self.sqlstate}): {self.query}: {self.msg}"
```

Comment:
I am not sure about this section. Should we add query here even when we were not printing sfqid

Reply:
I also think we should add sfqid. I don't know why we should hide them at WARN and ERROR level. That means user's production long won't have them.
But @sfc-gh-mkeller once mentioned that this should be regarded as a break change.